test/Parse/enum.swift

```diff
@@ -364,6 +364,13 @@ enum DuplicateMembers7 : String { // expected-error {{'DuplicateMembers7' declar
   case Foo = "Bar" // expected-error {{invalid redeclaration of 'Foo'}}
 }
 
+enum DuplicateMembers8 : String { // expected-error {{'DuplicateMembers8' declares raw type 'String', but does not conform to RawRepresentable and conformance could not be synthesized}}
+  case Foo // expected-note {{'Foo' previously declared here}}
+  // expected-note@-1 {{raw value previously used here}}
+  case Foo // expected-error {{invalid redeclaration of 'Foo'}}
+  // expected-error@-1 {{raw value for enum case is not unique}}
```

Comment:
Can you also test a couple of edge cases?

```
case Foo = "Foo"
case Foo = "Foo"
```
and
```
case Foo
case Bar = "Foo"
```

Reply:
the second one is already tested in `enum DuplicateMembers7` !

Reply:
I'll add a test for your first suggestion